python/ray/autoscaler/autoscaler.py

```diff
@@ -374,13 +382,14 @@ def target_num_workers(self):
 
     def launch_config_ok(self, node_id):
         launch_conf = self.provider.node_tags(node_id).get(
-            TAG_RAY_LAUNCH_CONFIG)
+            self.provider.tag_keys['launch-config'])
```

Comment:
nit: prefer double quotes by default, here and elsewhere

Reply:
Yeah, I noticed this a bit too late and just decided to write all this stuff with single quotes. I'll run all the files through yapf or something when this is closer to being ready.